tree/ntuple/v7/inc/ROOT/libdaos_mock/daos.h

```diff
@@ -90,15 +91,16 @@ typedef struct daos_event {
 
 //////////////////////////////////////////////////////////////////////////////// daos_event.h
 
-
 int daos_eq_create(daos_handle_t *eqh);
 int daos_eq_destroy(daos_handle_t eqh, int flags);
-int daos_eq_poll(daos_handle_t eqh, int wait_running,
-	     int64_t timeout, unsigned int nevents, daos_event_t **events);
+int daos_eq_poll(daos_handle_t eqh, int wait_running, int64_t timeout, unsigned int nevents, daos_event_t **events);
+int daos_eq_query(daos_handle_t eqh, daos_eq_query_t query, unsigned int nevents, daos_event_t **events);
+
+int daos_event_test(daos_event_t *ev, int64_t timeout, bool *flag);
+int daos_event_parent_barrier(daos_event_t *ev);
```

Comment:
I guess these new declarations are required by the follow-up PR that includes changes in the use of EQs, right?

Reply:
Correct, these should have only been added in the next PR. Sorry about the spoilers!